README.md

```diff
@@ -0,0 +1,38 @@
+# awx
+
+This role sets up a local AWX installation using the Docker installation.
+Non-containerized installation is not supported by AWX as of writing.
+
+The data can be found in `/var/lib/awx`.
+
+## Requirements
+
+Ubuntu
+
+## Role Variables
+
+| Name                | Default/Required    | Description                                      |
+|---------------------|:-------------------:|--------------------------------------------------|
+| `awx_arch`          | `amd64`             | Architecture of the target CPU                   |
+| `awx_postgres_data` | `/var/lib/postgres` | Location of the data of the PostgreSQL container |
+| `awx_tunables`      |                     | Dict of all installer tunables                   |
```

Comment:
Please explain or link to the tunables.

Reply:
There is currently no documentation on them :/

Reply:
Somehow you found them, you could link to the source of that or explain some of them.